src/main/java/com/volmit/iris/core/decrees/DecreeStudioExtension.java

```diff
@@ -0,0 +1,38 @@
+package com.volmit.iris.core.decrees;
```

Comment:
Put this in DecStudio, dont make extra classes just for studio.

Reply:
Four